aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/utils/SemVerUtils.java

```diff
@@ -7,7 +7,7 @@ private SemVerUtils() {
     }
 
     public static VersionInformation getMinCompatibleVersion() {
-        return VersionInformation.fromVersionString("2.8.3-funcrel");
+        return VersionInformation.fromVersionString("2.9.0-funcrel");
```

Comment:
Is this checked somewhere or is this just to be displayed somewhere ?

Reply:
Before triggering any command (even from JENKINS) a compatibility check if performed using this particular version.
Console-Tools can then target any CAST Imaging Console starting this version